terraform/geomatch_app/cognito/README.md

```diff
@@ -0,0 +1,122 @@
+
+## Cognito Module Variables
+
+- **project**: (string) The name of the project.
+- **environment**: (string) The environment (e.g., dev, prod).
+- **aws_region**: (string) The AWS region where resources are created.
+- **ecr_name_suffix**: (string) Default is "app". Suffix for the ECR name.
+- **idp_metadata_url**: (string) Default is "app". Metadata URL for the identity provider, e.g. https://login.stanford.edu/metadata.xml.
+- **ssm_name_prefix**: (string) Should be '/{project}/{environment}'.
+- **cognito_region**: (string) The region where the user pool is created.
+- **cognito_redirect_uri**: (list of strings) The redirect URI of the user pool, first element contains the relevant uri.
+- **cognito_email_verification_message**: (string) Email verification message.
+- **cognito_email_verification_subject**: (string) Email verification subject.
+- **cognito_admin_create_user_message**: (string) Admin create user message.
+- **cognito_admin_create_user_subject**: (string) Admin create user subject.
+- **cognito_allow_email_address**: (string) Allow email address for Cognito.
+- **subdomain**: (string) Subdomain for the application (e.g., 'us').
+- **domain**: (string) The domain for the Cognito callback URLs.
+- **external_providers**: (list of objects) List of external identity providers to be added to the Cognito user pool. Each object includes:
+  - **provider_name**: (string) Name of the provider.
+  - **provider_type**: (string) Type of the provider.
+  - **metadata_url**: (string) Metadata URL for the provider.
+  - **attribute_mapping**: (map of strings) Attribute mapping for the provider.
+  - **identifiers**: (list of strings) Identifiers for the provider.
+  - **sign_out_flow**: (bool) Whether to use sign out flow.
+  - **sign_saml_requests**: (bool) Whether to sign SAML requests.
+  - **require_encrypted_assertions**: (bool) Whether to require encrypted assertions.
+
+## Outputs
+
+- **cognito_region**: The AWS region where Cognito resources are created.
+- **cognito_client_id**: The Cognito User Pool Client ID.
+- **cognito_user_pool_id**: The Cognito User Pool ID.
+- **cognito_client_secret**: The Cognito User Pool Client Secret (sensitive).
+- **cognito_redirect_uri**: The Cognito redirect URI.
+- **cognito_app_domain**: The Cognito app domain.
+- **cognito_authorization_endpoint**: The Cognito authorization endpoint.
+- **cognito_token_url**: The Cognito token get endpoint.
+- **cognito_user_pool_arn**: The Cognito User Pool ARN.
+
+## Configuration Notes
+
+- **Stanford iDP Configuration**: By default, the Stanford iDP is configured to require encryption. However, this terraform module does not do a great job of enabling encryption by default.Users must manually enable the checkboxes for "Sign SAML requests to this provider" and "Require encrypted SAML assertions from this provider" in the Cognito console.
+
+## Resources
+
+### User Pool
```

Comment:
I think we should delete all the info about the resources, but I like the idea of listing the resources (user pool, provider, etc)